docs/specifications/tests/Zone-TP/zone11.md

```diff
@@ -0,0 +1,284 @@
+Z11_INCONSISTENT_SPF_POLICIES   | WARNING |                               | The *Child Zone* publishes different SPF policies on two different name servers.
+Z11_NO_SPF_FOUND                | DEBUG   |                               | The *Child Zone* does not publish an SPF policy.
+Z11_SPF1_HAS_P_MACRO            | WARNING | ns_ip_list, spf_policy        | The following SPF version 1 policy contains a %{p} macro, which should no longer be used: {spf_policy}. Policy retrieved from the following nameservers: {ns_ip_list}.
+Z11_SPF1_HAS_PTR                | WARNING | ns_ip_list, spf_policy        | The following SPF version 1 policy contains a ptr mechanism, which should no longer be used: {spf_policy}. Policy retrieved from the following nameservers: {ns_ip_list}.
+Z11_SPF1_MULTIPLE_RECORDS       | ERROR   | ns_ip_list                    | The *Child Zone* publishes more than one SPF version 1 policy. Policy retrieved from the following nameservers: {ns_ip_list}.
+Z11_SPF1_MULTIPLE_SAME_MODIFIER | ERROR   | ns_ip_list, spf_modifier      | The following SPF version 1 policy contains more than one {spf_modifier} modifier: {spf_policy}. Policy retrieved from the following nameservers: {ns_ip_list}.
+Z11_SPF1_RECURSIVE              | NOTICE  | ns_ip_list, spf_policy        | The following SPF version 1 policy uses "include" or "redirect": {spf_policy}. Policy retrieved from the following nameservers: {ns_ip_list}.
+Z11_SPF1_REDIRECT_AND_ALL       | WARNING | ns_ip_list, spf_policy        | The following SPF version 1 policy contains both the "all" keyword and a redirect modifier, which will cause "all" to be ignored: {spf_policy}. Policy retrieved from the following nameservers: {ns_ip_list}.
+Z11_SPF1_REDIRECT_NOT_AT_END    | WARNING | ns_ip_list, spf_policy        | The following SPF version 1 policy contains a redirect modifier, but it does not appear at the end of the string: {spf_policy}. Policy retrieved from the following nameservers: {ns_ip_list}.
+Z11_SPF1_SYNTAX_ERROR           | ERROR   | ns_ip_list, spf_policy        | The following SPF version 1 policy has a syntax error: {spf_policy}. Policy retrieved from the following nameservers: {ns_ip_list}.
+Z11_SPF1_SYNTAX_OK              | INFO    |                               | The *Child Zone*’s SPF version 1 policy has correct syntax.
+Z11_SPF1_TOO_COMPLEX            | ERROR   | ns_ip_list, spf_policy, count | Evaluating the following SPF version 1 policy would require {count} recursive DNS lookups by an SPF validator, exceeding the limit of 10: {spf_policy}. Policy retrieved from the following nameservers: {ns_ip_list}.
+Z11_SPF_WRONG_VERSION           | WARNING | ns_ip_list, spf_version       | The *Child Zone* publishes an SPF policy with invalid version number {spf_version}. Policy retrieved from the following name servers: {ns_ip_list}.
+Z11_UNABLE_TO_CHECK_SPF         | ERROR   |                               | Unable to check whether *Child Zone* publishes an SPF policy.
```

Comment:
"Z11_UNABLE_TO_CHECK_SPF" sounds more like "unable to validate SPF". Maybe "Z11_UNABLE_TO_CHECK_FOR_SPF"? On the other hand it seems less likely that a name sever responds authoritatively on a SOA record query but not an TXT record query.

Reply:
Okay, renaming the message tag.